test/correctness/simd_op_check.cpp

```diff
@@ -1323,6 +1323,23 @@ void check_hvx_all() {
     check("valign(v*,v*,r*)", hvx_width/1, in_u16(x + 4));
     check("valign(v*,v*,r*)", hvx_width/1, in_u16(x + hvx_width - 4));
 
+    check("vunpack(v*.ub)", hvx_width/1, u16(u8_1));
+    check("vunpack(v*.ub)", hvx_width/1, i16(u8_1));
+    check("vunpack(v*.uh)", hvx_width/2, u32(u16_1));
+    check("vunpack(v*.uh)", hvx_width/2, i32(u16_1));
+    check("vunpack(v*.b)", hvx_width/1, u16(i8_1));
+    check("vunpack(v*.b)", hvx_width/1, i16(i8_1));
+    check("vunpack(v*.h)", hvx_width/2, u32(i16_1));
+    check("vunpack(v*.h)", hvx_width/2, i32(i16_1));
+
+    check("vunpack(v*.ub)", hvx_width/1, u32(u8_1));
+    check("vunpack(v*.ub)", hvx_width/1, i32(u8_1));
+    check("vunpack(v*.b)", hvx_width/1, u32(i8_1));
+    check("vunpack(v*.b)", hvx_width/1, i32(i8_1));
+
+#if 0
```

Comment:
Yeah. I don't think it's possible to test vzxt/vsxt without a multi-stage pipeline that stores an intermediate buffer to memory. Without that, anything you write will get rewritten to a vunpack :)